feature/funding/googleplay/src/main/res/values/constants.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We enabled the funding feature for K-9 Mail. These links need to be updated in the K-9 Mail `constants.xml`.

Reply:
Done